spec/factories.rb

```diff
@@ -5,4 +5,10 @@
     title 'Test'
     target_date '01/01/2020'
   end
+
+  factory :user do
+    fullname 'Test'
+    email 'email@email.com'
+    # password 'password'
```

Comment:
If the factory doesn't need a password, go ahead and remove the comment.

Reply:
This comment was removed